ras_backstage/resources/party/update_respondent_details.py

```diff
@@ -0,0 +1,38 @@
+import logging
+
+from flask import request
+from flask_restplus import Resource, fields
+from structlog import wrap_logger
+
+from ras_backstage import party_api
+from ras_backstage.controllers import party_controller
+
+
+logger = wrap_logger(logging.getLogger(__name__))
+
+respondent_details = party_api.model('RespondentDetails', {
+    'respondent_id': fields.String(required=True),
+    'first_name': fields.String(required=True),
+    'last_name': fields.String(required=True),
+    'telephone': fields.String(required=True),
+})
+
+
+@party_api.route('/update-respondent-details')
+class UpdateRespondentDetails(Resource):
+    @staticmethod
+    @party_api.expect(respondent_details, validate=True)
+    def put():
+
+        logger.info('Retrieving updated respondent details')
```

Comment:
I'd probably parse `respondent_id` here first then include it in the log lines so we have a bit more context on these messages